most-used-words.el

```diff
@@ -50,97 +50,170 @@
 
 (require 'cl-lib)
 
+(defgroup most-used-words nil
+  "Display most used words in buffer."
+  :prefix "most-used-words-"
+  :group 'tool
+  :link '(url-link :tag "Repository" "https://github.com/udyantw/most-used-words"))
+
+(defcustom most-used-words-word-display 3
+  "Default word data to display."
+  :type 'integer
+  :group 'most-used-words)
+
+(defcustom most-used-words-display-type 'buffer
+  "The method to display most used words."
+  :type '(choice (const :tag "buffer" buffer)
+                 (const :tag "table" table))
+  :group 'most-used-words)
+
+(defvar most-used-words--buffer-name "*Most used words*"
+  "Buffer name for displaying most used words.")
```

Comment:
I have this on top, so we can use buffer name consistently everywhere in this package.

Reply:
Looks fine to me. I'm trusting your work: I've never used the DEFGROUP and DEFCUSTOM constructs.

Reply:
You should try it out though! For me, this just another way to declare variable but with different semantic meaning. 😄 Resolve this conversation for now!